dashboard/test/models/script_test.rb

```diff
@@ -203,10 +203,10 @@ def setup
 
   test 'get_script_level_by_stage_and_position raises RecordNotFound instead of NoMethodError when not found' do
     artist = Script.find_by_name('artist')
-    assert_raises(ActiveRecord::RecordNotFound) do
+#    assert_raises(ActiveRecord::RecordNotFound) do
```

Comment:
Ha, whoops, I commented out to let it continue so I could debug why it was failing but got distracted before the actually debugging part.

> On Jan 15, 2015, at 21:06, Josh Lory <notifications@github.com> wrote:
> 
> In dashboard/test/models/script_test.rb:
> 
> > @@ -203,10 +203,10 @@ def setup
> >  
> >    test 'get_script_level_by_stage_and_position raises RecordNotFound instead of NoMethodError when not found' do
> >      artist = Script.find_by_name('artist')
> > -    assert_raises(ActiveRecord::RecordNotFound) do
> > +#    assert_raises(ActiveRecord::RecordNotFound) do
> This test no longer asserts anything -- should it be removed?
> 
> —
> Reply to this email directly or view it on GitHub.
>